src/Command/Dump.php

```diff
@@ -66,7 +66,7 @@ protected function execute(InputInterface $input, OutputInterface $output): void
 
         $data = [
             'count' => $input->getOption('limit'),
-            'requeue' => !$input->getOption('no-requeue'),
+            'ackmode' => $input->getOption('ackmode'),
```

Comment:
should we validate that its one of the allowed values? if the rabbitmq call fails with a clear error i am fine leaving validation to rabbitmq, but if its ignored or a cryptic error, we should validate here to handle typos/completely invalid values.

Reply:
I was thinking the same, but felt the error from the rabbitmq API which is output is pretty clear.